package.json

```diff
@@ -2,14 +2,20 @@
   "name": "selfkey-ui",
   "version": "1.0.0",
   "description": "UI components for the SelfKey project",
-  "main": "index.js",
+  "main": "build/lib/index.js",
+  "types": "build/lib/index.d.ts",
+  "files": [
+    "build/lib"
+  ],
   "scripts": {
     "start": "start-storybook -p 6006",
-    "build": "npm run lint && npm run build-lib && build-storybook",
```

Comment:
@dfcowell I removed it because we don't need to wait to build the storybook if we are not going to push it to git. Or will we? I think the build of the storybook is for deploying it to a server or gh-pages, isn't it? Cheers